lib/internal/http.js

```diff
@@ -49,7 +49,7 @@ function emitStatistics(statistics) {
 }
 
 module.exports = {
-  outHeadersKey: Symbol('outHeadersKey'),
+  kOutHeaders: Symbol('outHeaders'),
```

Comment:
```suggestion
  kOutHeaders: Symbol('kOutHeaders'),
```

No written rules for this as well but this seems to be the most used recently

https://github.com/nodejs/node/blob/b4f0a18b5a717b10d2d1c98e7aba85f0ac33aa68/lib/internal/worker/io.js#L28-L35